lib/ansible/module_utils/network/common/facts/facts.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The junos facts base class
+this contains methods common to all facts subsets
+"""
+from ansible.module_utils.network.common.network import get_resource_connection
+from ansible.module_utils.six import iteritems
+
+
+class FactsBase(object):
+    """
+    The facts base class
+    """
+    def __init__(self, module):
+        self._module = module
+        self._warnings = []
+        self._gather_subset = module.params.get('gather_subset')
+        self._gather_network_resources = module.params.get('gather_network_resources')
+        self._connection = get_resource_connection(module)
+
+        self.ansible_facts = {'ansible_network_resources': {}}
+        self.ansible_facts['gather_network_resources'] = list()
+        self.ansible_facts['gather_subset'] = list()
```

Comment:
Change this to `self.ansible_facts['ansible_net_gather_subset'] = list()` as it is `ansible_net_gather_subset` in legacy facts module.

Reply:
Should `gather_network_resources` be changed to `ansible_gather_network_resources` to be in sync with the naming convention?

Reply:
+1